daemon/exec.go

```diff
@@ -262,6 +261,9 @@ func (d *Daemon) ContainerExecStart(name string, stdin io.ReadCloser, stdout io.
 		}
 		return nil
 	case err := <-execErr:
+		if err := <-attachErr; err != nil {
+			logrus.Warn(err)
```

Comment:
If the exec is ok but the attach fails, is it really ok to just silently ignore the attach error?  While we log it to the daemon, the end user won't see it.  I'm wondering if we should halt processing instead.